packages/fxa-settings/src/models/integrations/oauth-redirect-integration.ts

```diff
@@ -0,0 +1,27 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
```

Comment:
nit: double license.

So I did a quick grep (naturally) to see if this happens in other files, but only found 2 results:
```sh
git grep -c "/* This Source Code Form is subject to the terms of the Mozilla Public" | grep -Ev ":1$"
packages/fxa-content-server/app/tests/spec/lib/cocktail.js:2
packages/fxa-settings/src/models/integrations/oauth-redirect-integration.ts:2
```

And the cocktail spec one is "interesting":
```sh
head -4 packages/fxa-content-server/app/tests/spec/lib/cocktail.js | cat -n

     1	/* This Source Code Form is subject to the terms of the Mozilla Public
     2	/* This Source Code Form is subject to the terms of the Mozilla Public
     3	 * License, v. 2.0. If a copy of the MPL was not distributed with this
     4	 * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
```